### PR TITLE
Add multi-sort on all Vue tables.

### PR DESCRIPTION
**Pull request by jschwartzentruber**
Ctrl+click to keep the existing sort and add another sort column.

### CODE REVIEW

**Comment by jschwartzentruber** [comment]
Fixes #769

File: server/frontend/src/components/Crashes/List.vue [hunk 1]
```diff
@@ -311,20 +359,17 @@ export default {
...
-        let hashReverse = false;
-        if (hashSortKey.startsWith("-")) {
-          hashSortKey = hashSortKey.substring(1);
-          hashReverse = true;
-        }
-        if (validSortKeys.includes(hashSortKey)) {
-          this.sortKey = hashSortKey;
-          this.reverse = hashReverse;
-        } else {
+        let sortKeys = hash.sort.split(",").filter((key) => {
```
Comment (pyoor) [inline server/frontend/src/components/Crashes/List.vue:362]: nit - use const

File: server/frontend/src/components/Signatures/List.vue [hunk 2]
```diff
@@ -202,20 +225,17 @@ export default {
...
-        let hashReverse = false;
-        if (hashSortKey.startsWith("-")) {
-          hashSortKey = hashSortKey.substring(1);
-          hashReverse = true;
-        }
-        if (validSortKeys.includes(hashSortKey)) {
-          this.sortKey = hashSortKey;
-          this.reverse = hashReverse;
-        } else {
+        let sortKeys = hash.sort.split(",").filter((key) => {
```
Comment (pyoor) [inline server/frontend/src/components/Signatures/List.vue:228]: nit - use const

Comment (pyoor) [inline server/frontend/src/components/Signatures/List.vue:228]:
```suggestion
        const sortKeys = hash.sort.split(",").filter((key) => {
```

**Review by pyoor — APPROVED** [review]
Approved with minor suggestions.

**Pull request merged**